### PR TITLE
Constraint version of WTForms to non-broken version

### PR DESCRIPTION
**Pull request by ashb**
**What this PR does / why we need it**:

**Special notes for your reviewer**:

### CODE REVIEW

File: 1.10.7/buster/include/pip-constraints.txt [hunk 1]
```diff
@@ -2,3 +2,5 @@
 
 redis!=3.4.0
 Werkzeug < 1.0.0
+
+WTforms<2.3.0
```
Comment (kaxil) [inline 1.10.7/buster/include/pip-constraints.txt:6]:
```suggestion
WTforms<2.3.0
```

File: 1.10.7/alpine3.10/include/pip-constraints.txt [hunk 2]
```diff
@@ -5,3 +5,5 @@ Werkzeug < 1.0.0
 # Make sure we don't get pyarrow installed, libarrow is difficult to get working in Alpine
 papermill<2.0
 nteract-scrapbook<0.4
+
+WTforms<2.3.0
```
Comment (kaxil) [inline 1.10.7/alpine3.10/include/pip-constraints.txt:9]:
```suggestion
WTforms<2.3.0
```

File: 1.10.6/buster/include/pip-constraints.txt [hunk 3]
```diff
@@ -3,3 +3,5 @@ azure-storage-blob<12.0
 pymssql<3.0
 redis!=3.4.0
 Werkzeug < 1.0.0
+
+WTforms<2.3.0
```
Comment (kaxil) [inline 1.10.6/buster/include/pip-constraints.txt:7]:
```suggestion
WTforms<2.3.0
```

File: 1.10.6/alpine3.10/include/pip-constraints.txt [hunk 4]
```diff
@@ -8,3 +8,5 @@ Werkzeug < 1.0.0
 # Make sure we don't get pyarrow installed, libarrow is difficult to get working in Alpine
 papermill<2.0
 nteract-scrapbook<0.4
+
+WTforms<2.3.0
```
Comment (kaxil) [inline 1.10.6/alpine3.10/include/pip-constraints.txt:12]:
```suggestion
WTforms<2.3.0
```

File: 1.10.5/rhel7/include/pip-constraints.txt [hunk 5]
```diff
@@ -3,3 +3,5 @@ azure-storage-blob<12.0
 pymssql<3.0
 redis!=3.4.0
 Werkzeug < 1.0.0
+
+WTforms<2.3.0
```
Comment (kaxil) [inline 1.10.5/rhel7/include/pip-constraints.txt:7]:
```suggestion
WTforms<2.3.0
```

File: 1.10.5/buster/include/pip-constraints.txt [hunk 6]
```diff
@@ -4,3 +4,5 @@ elasticsearch>=5.5.3,<6.0.0
 pymssql<3.0
 redis!=3.4.0
 Werkzeug < 1.0.0
+
+WTforms<2.3.0
```
Comment (kaxil) [inline 1.10.5/buster/include/pip-constraints.txt:8]:
```suggestion
WTforms<2.3.0
```

File: 1.10.5/alpine3.10/include/pip-constraints.txt [hunk 7]
```diff
@@ -8,3 +8,5 @@ Werkzeug < 1.0.0
 # Make sure we don't get pyarrow installed, libarrow is difficult to get working in Alpine
 papermill<2.0
 nteract-scrapbook<0.4
+
+WTforms<2.3.0
```
Comment (kaxil) [inline 1.10.5/alpine3.10/include/pip-constraints.txt:12]:
```suggestion
WTforms<2.3.0
```

File: 1.10.10.dev/buster/include/pip-constraints.txt [hunk 8]
```diff
@@ -1,3 +1,5 @@
 # Constraint the version pip will install, if it ever needs to install a module
 
 redis!=3.4.0
+
+WTforms<2.3.0
```
Comment (kaxil) [inline 1.10.10.dev/buster/include/pip-constraints.txt:5]:
```suggestion
WTforms<2.3.0
```

File: 1.10.10.dev/alpine3.10/include/pip-constraints.txt [hunk 9]
```diff
@@ -4,3 +4,5 @@ redis!=3.4.0
 # Make sure we don't get pyarrow installed, libarrow is difficult to get working in Alpine
 papermill<2.0
 nteract-scrapbook<0.4
+
+WTforms<2.3.0
```
Comment (kaxil) [inline 1.10.10.dev/alpine3.10/include/pip-constraints.txt:8]:
```suggestion
WTforms<2.3.0
```

File: 1.10.6/alpine3.10/include/pip-constraints.txt (hunk 4 above)
Comment (ashb) [reply 1.10.6/alpine3.10/include/pip-constraints.txt:12]: The reason I added a new line here is the comment on L8 applies to th next two lines, but this is a separate "group"

**Pull request merged**